InnerEyeAMLPLDeployment/README.md

```diff
@@ -10,6 +10,15 @@ With some limitations listed below, we were able to successfully perform the fol
 
 ![AML in a closed VNet](./assets/PrivateAML.png)
 
+### Pre-Requisites
```

Comment:
These are the steps to install the required tools on a Mac. Terraform, Azure CLI will be installed differently on Windows.

Reply:
@javier-alvarez , please update this heading or section to highlight that these are the steps on a Mac.  Also please check md linting error (don't use bare urls)